ansi2html/converter.py

```diff
@@ -786,6 +787,8 @@ def main() -> None:
         title=opts.output_title,
     )
 
+    reader = io.TextIOWrapper(sys.stdin.buffer, opts.input_encoding, "replace")
```

Comment:
I switched from `sys.stdin.detach()` to `sys.stdin.buffer`, because it originally failed with `"TextIO" has no attribute "detach"`.

The old approach with catching an exception in production code just to make the tests work does smell a bit. And also 
```python
try:

except Error: 
    pass
```
is an anti-pattern that should not have been used in the first place.